README.rst

```diff
@@ -102,7 +106,13 @@ using:
 
    user@host:~$ gpg --export <long_key_ID> > <long_key_ID>
 
-And then specify key ID to person mapping in the ``keyids`` file.
+And then specify the key ID to person mapping in the ``keyids`` file.
+
+Then please state in the ``git`` commit messages how confident you are that `no
+one has ever had a copy of your private key
+<https://www.wired.com/2014/10/snowdens-first-emails-to-poitras/>`_ and
+optionally, in case you feel like disclosing parts of your OpSec, the
+measures you implemented to prevent this from happening.
```

Comment:
[This article](https://www.debian.org/devel/join/nm-step2.en.html) seems to be relevant. In short, to become a Debian Developer, an Applicant is required to provide a key signed by two existing Debian Developers. Therefore, everything is based on the GPG Web of Trust. I don't think Debian requires disclosing your GPG key practices to the public, at least it doesn't seem that way.

I would still remove this paragraph from the readme, it seems unnecessary to me. I expect that the GPG keys won't be added to this repository willy-nilly, some kind of vetting process would be required - GPG key should be used by the person providing it for some time, preferably to sign code, it should have strong presence on [keybase.io](https://keybase.io/) meaning that the proof of ownership should come from multiple accounts that contain some history. Any signatures on the GPG key add to its value as well. I don't plan to add GPG keys left an right. :-)